test/unit/postprocessing/test_curve_utilities.py

```diff
@@ -89,24 +89,37 @@ def test_calculate_roc_points():
     grouped_data = data.groupby(SENSITIVE_FEATURE_KEY).get_group("A") \
         .sort_values(by=SCORE_KEY, ascending=False)
 
-    roc_points = _calculate_roc_points(grouped_data, "A")
+    roc_points = _calculate_roc_points(grouped_data, "A", flip=True)
     expected_roc_points = pd.DataFrame({
-        "x": [0, 0.25, 0.5, 0.5, 1],
-        "y": [0, 1/3,  2/3, 1,   1],
+        "x": [0, 1, 0.75, 0.25, 0.5, 0.5, 0.5, 0.5, 1, 0],
+        "y": [0, 1, 2/3,  1/3,  1/3, 2/3, 0,   1,   1, 0],
         "operation": [ThresholdOperation('>', np.inf),
+                      ThresholdOperation('<', np.inf),
```

Comment:
Why change the original example?

Reply:
Keeping the existing tests means we didn't regress anywhere so that would be very important for a PR of this size. You can add as many additional examples as you can write :-)

Reply:
The logic of `_calculate_roc_points` has changed--that's why the change here. New logic is more generic, so I think I prefer it.

Reply:
I'm actually not entirely sure whether I understand it anymore. These used to be ordered in an understandable way, but now it's hard to grasp how this all came to be.

You mean the old example doesn't work anymore? Because it's hard to tell if we break something if we change the test logic itself.

Reply:
[exactly... let me re-order them in an understandable way again]

Reply:
OK, thanks for elaborating!
Would it be possible to keep the old case (probably with adjusted result) and add this new case additionally?

Reply:
Technically, the behavior of `_calculate_roc_points` is now different (it produces lists twice as long as before), so this is a breaking change. If I left the previous test unmodified it would fail. What I have here is an equivalent test (on the exactly same input data). Do you want to separate out the old functionality of the previous  `_calculate_roc_points`? (I'm not sure I see why.)

Reply:
I feel like we already agree, but I'm expressing myself badly :-)
What I meant to say is that when you change logic like in `_calculate_roc_points`, then it's natural that the test case expectations may need to be adjusted. For example, the expected result may be twice as long as you mention. However, in this particular case the input data itself was also changed (or at least it looks like that to me), which means it's not actually possible for us to understand how it affects the test case without calculating everything by hand. From the diff I expected to see the identical input data and then an expected result list that's twice as long.
Perhaps git/GitHub is playing tricks on me and you didn't change the test input data?

Reply:
Please take a look again. To make it clear how I changed things, the additional returned points are written in the form `.0` and `.5` rather than `0` and `0.5` etc.